AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -42,38 +42,36 @@ void RayTraceCorrelator::SetAngularConfig(double angularSize){
     angularSize_ = angularSize;
     numPhiBins_ = int(360. / angularSize);
     numThetaBins_ = int(180. / angularSize);
-
-    // now fill up the phi and theta angles
-    double PhiWaveDeg, ThetaWaveDeg;
-    for (int i = 0; i < numPhiBins_; i++) {
-        PhiWaveDeg = -180 + 0.5 * angularSize_ + angularSize_ * i;
-        phiAngles_.push_back(PhiWaveDeg * TMath::DegToRad());
+    this->internalMap = std::unique_ptr<TH2D>(new TH2D("","",numPhiBins_, -180, 180, numThetaBins_, -90, 90));
+
+    // start loop at 0, but ask for GetBinCenter(i+1)
+    // This is not an accident!
+    // This is because ROOT documentation says bin=0 is underflow
+    // See https://root.cern.ch/doc/master/classTH1.html#ac70dcb4082bd510f5aa3e6efeb043f39
+    for(int i=0; i<this->internalMap->GetNbinsX(); i++){
+        phiAngles_.push_back(this->internalMap->GetXaxis()->GetBinCenter(i+1));
     }
-    for (int i = 0; i < numThetaBins_; i++) {
-        ThetaWaveDeg = -90 + 0.5 * angularSize_ + angularSize_ * i;
-        thetaAngles_.push_back(ThetaWaveDeg * TMath::DegToRad());
+    for(int i=0; i<this->internalMap->GetNbinsY(); i++){
+        thetaAngles_.push_back(this->internalMap->GetYaxis()->GetBinCenter(i+1));
     }
 }
 
-void RayTraceCorrelator::ConvertAngleToBins(double theta, double phi, 
-    int &thetaBin, int &phiBin
-    ){
+int RayTraceCorrelator::ConvertAngleToTH2DGlobalBin(double theta, double phi){
 
-    if(abs(theta) > 91 || isnan(theta)){
+    if(abs(theta) > 90 || isnan(theta)){
```

Comment:
Why is this just >90? Why not >=90?

Reply:
I think ==90 is actually ok? Like, if the upper bin edge is inclusive, then this is okay, right? Or am I underthinking it?

Reply:
Oh yes- I misread it. All good here!

Reply:
Okay, thanks!